src/lib/index.ts

```diff
@@ -187,6 +187,7 @@ export const GridRow = styled('div', {
 export interface GridColumnProps {
   display?: DisplayProp
   width?: WidthProp
+  offset?: WidthProp
```

Comment:
Could you add the "offset" to the shouldForwardProp callback? This prop now gets added as a HTML attribute.

So:

`shouldForwardProp: (prop) => isPropValid(prop) && prop !== 'width' && prop !== 'display' && prop !== 'offset'`